### PR TITLE
Php tests

### PR DESCRIPTION
**Pull request by rimas-kudelis**
This adds PHP tests for #3626, #3627, and #3629.

### CODE REVIEW

File: translate/storage/test_php.py [hunk 1]
```diff
@@ -461,6 +461,23 @@ def test_parsing_arrays_with_space_before_array_declaration(self):
         assert phpunit.name == "$lang->'item2'"
         assert phpunit.source == "value2"
 
+    @mark.xfail(reason="Bug #3629")
+    def test_parsing_arrays_declared_in_a_single_line(self):
+        """parse an array declared in a single line.
+        Bug #3629"""
+        phpsource = '''$lang = array( 'item1' => 'value1', 'item2' => 'value2', 'item3' => 'value3' );'''
+        phpfile = self.phpparse(phpsource)
+        assert len(phpfile.units) == 2
```
Comment (unho) [inline translate/storage/test_php.py]: There are 3 entries in the array, not 2.

File: translate/storage/test_php.py [hunk 2]
```diff
@@ -617,6 +651,31 @@ def test_parsing_nested_arrays_with_blank_entries(self):
...
+        """ignore the /* comment delimiter when it is part of a string.
+        Bug #3627"""
+        phpsource = '''$definition['key'] = 'Value /* value continued';
+      $lang = array(
+         'somekey' => 'Some value',
+         'otherkey' => 'Other value /* continued',
+         'thirdkey' => 'Third value',
+      );'''
+        phpfile = self.phpparse(phpsource)
+        assert len(phpfile.units) == 2
```
Comment (unho) [inline translate/storage/test_php.py]: There are 3 entries in this other array too.

Comment (rimas-kudelis) [inline translate/storage/test_php.py]: Actually, I think it's four.

**Comment by unho** [comment]
Looks ok, but would be great to include tests for https://github.com/translate/translate/issues/3628 too.

**Comment by rimas-kudelis** [comment]
That has already been added by 13f9ea0325ae929688fbf46482d021ebbb88ee4d, I think.

**Comment by unho** [comment]
@rimas-kudelis Merged. Thank you very much for your contribution.

**Pull request merged**